internal/api/event.go

```diff
@@ -1,50 +1,36 @@
 package api
 
 import (
+	"context"
 	"fmt"
 	"time"
 
-	"github.com/go-openapi/strfmt"
+	v1 "github.com/metal-stack/metal-api/pkg/api/v1"
 	"github.com/metal-stack/metal-core/pkg/domain"
-	"github.com/metal-stack/metal-go/api/client/machine"
-	"github.com/metal-stack/metal-go/api/models"
+	"google.golang.org/protobuf/types/known/timestamppb"
 
 	"go.uber.org/zap"
 )
 
-func (c *apiClient) AddProvisioningEvent(machineID string, event *models.V1MachineProvisioningEvent) error {
+func (c *apiClient) AddProvisioningEvent(machineID string, event *v1.EventServiceSendRequest) error {
```

Comment:
This method doesn't exist anymore in the interface APIClient. Do we really need it?

Reply:
right,